src/main/java/com/mhcure/logmerge/filereader/MhFileReader.java

```diff
@@ -103,7 +103,7 @@ public Map<Long, String> readFileUsingBufferedReader(String fileName) throws Exc
         boolean isEncryptedFile = false;
         Cipher cipherObject = null;
         if (!logFile.isFile()) {
-            System.out.println(fileName + MhMessagePropertiesFileReader.getMessage(MhMessageKeyEnum.IGNORE_INVALID_FILE.getKey()));
+            MhFileAggregatorHelper.printToConsole(fileName + MhMessagePropertiesFileReader.getMessage(MhMessageKeyEnum.IGNORE_INVALID_FILE.getKey()));
             return fileContentsMap;
         }
```

Comment:
at line no 119 the variable lineCounter is unused .

Reply:
removed